### ISSUE TITLE
"Double-quoted" in 7.3.0 version

### ISSUE THREAD

**Issue by LACHKAR99ZAKARIA** [firebase/firebase-ios-sdk#12111]
### Description

"Double-quoted" in 7.3.0 version #12111

### Reproducing the issue

Hello, I have encountered the same issue as described in this post, but in version 7.3.0 on Xcode 15 and cocoapods 1.8.4 version. I would like to know how to resolve this problem while maintaining the current version. Here is the link to the relevant GitHub issue: https://github.com/firebase/firebase-ios-sdk/issues/5987

### Firebase SDK Version

7.3.0

### Xcode Version

15

### Installation Method

Swift Package Manager

### Firebase Product(s)

AB Testing, Crashlytics, Firestore

### Targeted Platforms

iOS

### Relevant Log Output

```shell
Double-quoted include "pb.h" in framework header, expected angle-bracketed instead
Double-quoted include "pb_common.h" in framework header, expected angle-bracketed instead
Double-quoted include "pb_decode.h" in framework header, expected angle-bracketed instead
Double-quoted include "pb_encode.h" in framework header, expected angle-bracketed instead
Double-quoted include "pb.h" in framework header, expected angle-bracketed instead
Double-quoted include "pb_decode.h" in framework header, expected angle-bracketed instead
Double-quoted include "pb_common.h" in framework header, expected angle-bracketed instead
Double-quoted include "pb.h" in framework header, expected angle-bracketed instead
Double-quoted include "pb_encode.h" in framework header, expected angle-bracketed instead
Double-quoted include "pb_common.h" in framework header, expected angle-bracketed instead
Double-quoted include "pb.h" in framework header, expected angle-bracketed instead
Double-quoted include "pb.h" in framework header, expected angle-bracketed instead
Double-quoted include "pb.h" in framework header, expected angle-bracketed instead
```

### If using Swift Package Manager, the project's Package.resolved

<details>
<summary>Expand <code>Package.resolved</code> snippet</summary>
<br>

```json

Replace this line with the contents of your Package.resolved.

```

</details>

### If using CocoaPods, the project's Podfile.lock

<details>
<summary>Expand <code>Podfile.lock</code> snippet</summary>
<br>

```yml

Replace this line with the contents of your Podfile.lock!

```

</details>

**Comment by rizafran** [comment]
Hi @LACHKAR99ZAKARIA, based on the related bug, it seems that the issue is on CocoaPods. You mentioned that you're using CocoaPods ver. 1.8.4 and as per checking, the issue was fixed on 1.10. With this, could you try to update your CocoaPods to 1.10 and see if the issue persists?

**Comment by LACHKAR99ZAKARIA** [comment]
I couldn't update my CocoaPods version due to a Ruby version mismatch. After encountering this issue, I found a solution on how to force update the Ruby version to 2.7.7 here: https://github.com/rvm/rvm/issues/5254. Subsequently, I installed the latest version of CocoaPods.
